requirements.functest.txt

```diff
@@ -10,12 +8,11 @@ git+https://github.com/openstack/neutron-lbaas.git@stable/liberty
 git+https://github.com/F5Networks/f5-openstack-lbaasv2-driver.git@liberty
 
 # Test utilities
-git+https://github.com/F5Networks/f5-openstack-test.git
 git+https://github.com/F5Networks/pytest-symbols.git
 
 mock==2.0.0
-pytest==3.0.3
-pytest-cov==2.3.1
+pytest>=3.0.5,<4
```

Comment:
Do we really need to update these requirements?  Are they giving us something we need?